Added service naming caveat and image to Full Host profiling enabling page (#30867)
DataDog/documentation: content/en/profiler/enabling/full_host.md

: Symbols should be available locally or can be uploaded in CI with `datadog-ci` 

## Installation

> <div class="alert alert-info">Always set <b>DD_SERVICE</b> for each service you want to profile and identify separately. This ensures accurate attribution and more actionable profiling data.</div>

Reviewer: ```suggestion

<div class="alert alert-info">Always set <b>DD_SERVICE</b> for each service you want to profile and identify separately. This ensures accurate attribution and more actionable profiling data. To learn more, see <a href="#service-naming">Service naming</a.</div>
```

   make
   ```

> ## Service Naming

Reviewer: ```suggestion
## Service naming
```

> When using full-host profiling, Datadog captures profiles from all processes running on the host. The service name for each process is primarily determined by the `DD_SERVICE` environment variable.

Reviewer: ```suggestion
When using full-host profiling, Datadog captures profiles from all processes running on the host. The service name for each process depends on the `DD_SERVICE` environment variable.
```

> If `DD_SERVICE` is set, the profiler will use the value of `DD_SERVICE` as the service name. This is the recommended and most reliable approach.

Reviewer: ```suggestion
If `DD_SERVICE` is set, the profiler uses the value of `DD_SERVICE` as the service name. This is the recommended and most reliable approach.
```

> If `DD_SERVICE` is not set, Datadog will infer a service name by using the binary name, which for interpreted languages is the name of the interpreter. So for services written in Java, full host will set service:`java` or service:`python3.12`

Reviewer: ```suggestion
If `DD_SERVICE` is not set, Datadog infers a service name from the binary name. For interpreted languages, this is the name of the interpreter. For example, for a service written in Java, the full-host profiler sets the service name to `service:java`.
```

{{< img src="profiler/inferred_service_example.png" alt="Example of an inferred services within Profiling" style="width:50%;">}}

> If multiple services are running under the same interpreter (for example, two separate Java applications on the same host), and neither sets `DD_SERVICE`, they will be grouped together under the same service name (e.g., service:java). Datadog cannot distinguish between them without a unique service name being provided.

Reviewer: ```suggestion
If multiple services are running under the same interpreter (for example, two separate Java applications on the same host), and neither sets `DD_SERVICE`, Datadog groups them together under the same service name. Datadog cannot distinguish between them unless you provide a unique service name.
```

> Always set `DD_SERVICE` for each service you want to profile and identify separately. This ensures accurate attribution and more actionable profiling data.

Reviewer: It would be best to remove this to avoid duplication.
```suggestion
```
